content/en/dashboards/guide/context-links.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Context Links
+kind: guide
+further_reading:
+- link: '/dashboards/widgets'
+  tag: 'Documentation'
+  text: 'List of Dashboard Widgets'
+
+---
+
+## Overview
+
+Your dashboards are fundamental in troubleshooting the root cause of an issue or determining an assertion to a critical question. 
+
+Dashboards can appear in your [notebooks][1], attached to a [monitor notification][2], or serve as a [screenboard][3]. You can start an investigation with a screenboard, document your insights in a notebook, and stay alerted by linking a monitor notification to your dashboard. 
+
+Use Datadog dashboards to perform an initial assessment of your context-relevant data sources and explore insights in Datadog or in a third-party application.  
+
+## Dashboard widgets
+
+Dashboard widgets contain default links to relevant Datadog pages with information about the scope including the timerange, underlying queries, and the series on a graph. [Generic widgets][4] contain context links, allowing you to point to third-party applications and override the default links to customize your landing page. 
+
+{{< img src="dashboards/guide/context_links/overview.mp4" alt="Context Link Demo" video="true" style="width:80%;">}}
+
+This guide introduces context links and provides examples of how to create external links and override default links.
+
+## Context links
+
+When investigating an issue, you may need to access external data or perform follow-up actions in a third-party application. Context links allow you to access and trigger third-party applications through webhooks in your dashboard widgets. 
```

Comment:
This is Overview section material.

By the way, the guide is about the custom link feature as a whole. Customisation of custom links is one important aspect of it, but only one aspect.

Reply:
Also, accessing / triggering third-part applications is one thing users might want to do, but likely not the only or most common one. Let's phrase accordingly, something like: 

After you've made an assessment in a Dashboard, your next step could be, for instance:
* dive deeper in the data, using specialised features in other Datadog pages (such as Explorers)
* broaden investigations, jumping to another Dashboard
* get insights or make actions from third-part applications